### ISSUE TITLE
no-duplicate-imports

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#48]
(optional?) rule to warn on more than one import from the same module path, i.e.

```js
import { x } from './mod'
import { y } from './mod'
```
reports on both lines, whereas

```js
import { x, y } from './mod'
```

is valid.

Also need to consider an option / internal work up / an additional rule to consider the abbreviated export/import syntax from Lee Byron's ES7 import proposal.

i.e. `no-import-export` or something to warn on 
```js
import { foo } from './mod'
export { foo }
```
and recommend
```js
export { foo } from './mod'
```

Those are valid ES6, but would need to consider things like 
```js
export MyCoolClass from './mod'
``` 
vs.
```js
import MyCoolClass from './mod'
export default MyCoolClass
```
 as the latter is the valid ES6 form.

**Comment by benmosher** [comment]
Also: may want to optionally allow
```js
import { x, y, z } from './mod'
import * as ns from './mod'
```
as there is no way to do this in one line.

That said, seems like it would generally be unintentional... unless the lines are adjacent?

**Comment by benmosher** [comment]
thought: use resolved module path as map key; current implementation does not flag aliased repeat imports:

```js
import { x } from './mod'
import { y } from 'src/mod'
```
